plugin/src/main/java/io/spine/tools/gradle/bootstrap/CodeGenExtension.java

```diff
@@ -40,7 +41,7 @@
 abstract class CodeGenExtension implements Logging {
 
     private final ProtobufGenerator protobufGenerator;
-    private final ProtocPlugin codeGenJob;
+    private final @Nullable ProtocPlugin codeGenJob;
```

Comment:
Can you please comment on this.

The PR description will get lost in our memories. But the field-level comments will not.

I would also consider `Optional` instead.